js/services/http.js

```diff
@@ -73,28 +12,6 @@ define(function(require, exports) {
       return !authProviders[url] && url.startsWith(config.api.url);
     }
 
-    /**
-     * Checks HTTP status for errors.
-     * @param  { {[x: string]: any} } response
-     * @return {boolean}
-     */
-    checkStatusError(response) {
-      const status = response.status;
-
-      switch (status) {
-        case STATUS.OK:
-          return true;
-        case STATUS.UNAUTHORIZED:
-          this.handleUnauthorized(response.json);
-          break;
-        default:
-          this.handleUnexpectedError();
-          break;
-      }
-
-      return false;
-    }
-
     sendRequest(method, path, payload) {
```

Comment:
Why was this method left but not inherited?

Reply:
@pavgra because of this line:
`headers: this.isSecureUrl(path) ? this.getHeaders() : {},`